RStudioProxy/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const morgan = require('morgan')
+
+const { createProxyMiddleware } = require('http-proxy-middleware')
+
+const { PORT, HOST } = require('./config')
+const { prepareInstancesMiddleware } = require('./init')
+
+const { instance, proxy, timers } = require('./service')
+
+// Create Express Server
+const app = express()
+
+app.use(morgan('dev'))
+
+// Info GET endpoint
+app.get('/prepare', prepareInstancesMiddleware, (req, res) => {
+  res.json({ status: 'OK' })
+})
+
+app.get('/timers', timers.timmersMiddleware, (req, res) => {
```

Comment:
This command show in console the active timmers and should be removed in the future